pyaqueduct/api.py

```diff
@@ -179,3 +180,53 @@ def get_extensions(self) -> List[Extension]:
             )
             for extension_data in self._client.get_extensions()
         ]
+
+    @validate_call
+    def get_task(
+        self,
+        task_id
+    ) -> Task:
+        """Get task by passing task_id.
+
+        Returns:
+            Task object
+        """
+        return Task(
+            client=self._client,
+            task_id=task_id
+        )
+
+    @validate_call
+    def get_tasks(  # pylint: disable=too-many-arguments, duplicate-code
+        self,
+        limit: PositiveInt = 10,
```

Comment:
Is this common behaviour? If I pass no limit I (Stas) assume I will be given all tasks. Is this the same in GQL behaviour? If not, please make them similar.

Reply:
Allowing User to get all the records present in backend can result in very large data getting passed to the client. We surely need to have limit for this. 

In case of Experiment service, we have a global variable `MAX_EXPERIMENTS_PER_REQUEST`, which validates that the value of limit to be less that the maximum limit, else it raises an exception with an error message.

We don't have such validation for Tasks in `query_schema..py`. I think we should add this validation in the backend.

As of this default value parameter, this is uniform with the `find_experiments` function defined above in this file.